sig-node/annual-report-2021.md

```diff
@@ -0,0 +1,72 @@
+# SIG Node Annual Report 2021
```

Comment:
I don't think we ever merged the report. Have we? I glanced over, it looks OK anyways.